tests/unit/lib/SGVector_unittest.cc

```diff
@@ -98,29 +98,6 @@ TEST(SGVectorTest,setget)
 	EXPECT_EQ(v[3], 0);
 }
 
-TEST(SGVectorTest,add)
-{
-	const int32_t seed = 17;
-
-	SGVector<float64_t> a(10);
-	SGVector<float64_t> b(10);
-
-	std::mt19937_64 prng(seed);
-	random::fill_array(a, 0.0, 1024.0, prng);
-	random::fill_array(b, 0.0, 1024.0, prng);
-	float64_t* b_clone = SGVector<float64_t>::clone_vector(b.vector, b.vlen);
-	SGVector<float64_t> c(b_clone, 10);
-
-	c.add(a);
-	for (int i=0; i < c.vlen; ++i)
-		EXPECT_EQ(c[i], a[i]+b[i]);
-
-	c = a + a;
-	EXPECT_EQ(c.vlen, 10);
-	for (int i=0; i < c.vlen; ++i)
-		EXPECT_EQ(c[i], 2*a[i]);
-}
-
 TEST(SGVectorTest,norm)
```

Comment:
future note: the norm should also be removed :)

Reply:
Yes, @karlnapf  I shall get to that asap once this PR gets merged :)